mypy/subtypes.py

```diff
@@ -666,13 +667,12 @@ def visit_unpack_type(self, left: UnpackType) -> bool:
         return False
 
     def visit_parameters(self, left: Parameters) -> bool:
-        if isinstance(self.right, (Parameters, CallableType)):
-            right = self.right
-            if isinstance(right, CallableType):
-                right = right.with_unpacked_kwargs()
```

Comment:
I can't tell why you removed support for callable types here. I guess that might be handled elsewhere?

Reply:
We should normally never compare `Callable` vs `Parameters`, and if we do they can never be subtype of each other, in the same sense as `int` is never a sub- (or super-) type of `(int) -> int`.